src/main.go

```diff
@@ -265,38 +351,50 @@ func process(data []models.DmwKnownContact, dfoAuthTokenObj models.DfoAuthTokenO
 			fmt.Printf("[%s] there were no contacts added to sanitizedUpdateRecords list - will not attempt to process updates\n", method)
 		}
 	}
+
+	// Create output file and print updated contacts to console
+	filepath := fmt.Sprintf("C:\\Users\\kristin.barlow\\ContactCloseUpdates\\%s_%s_%d.csv", inputData.Cluster, inputData.BusNo, time.Now().UnixNano())
+
+	mr, _ := json.Marshal(sanitizedUpdateRecords)
+	err := os.WriteFile(filepath, mr, 0644)
+	check(err)
+
+	for _, record := range sanitizedUpdateRecords {
+		fmt.Println(record)
+	}
+}
+
+func check(e error) {
```

Comment:
I would make this name more specific.  func check doesn't exactly say what it checks.

Reply:
I just removed that and put it inside the helper writeLogFile func.